RamsaySt.roboFontExt/lib/ramsaySt.py

```diff
@@ -73,13 +74,17 @@ def setGlyph(self, glyph):
 
     def glyphEditorDidSetGlyph(self, info):
         self.setGlyph(info["glyph"])
+        self.setAdjunctObjectsToObserve([self.leftGlyph, self.rightGlyph])
```

Comment:
move this to the `setGlyph` if setGlyph is called elsewhere those glyph are also observed as adjuncts